feature_server/config/scripts/spadenadefix.py

```diff
@@ -0,0 +1,17 @@
+"""
+spadenadefix.py - blocks spade-nade bug
+by kmsi(kmsiapps@gmail.com)
+version 1(2017.01.21)
+"""
+def apply_script(protocol,connection,config):
+
+    class kmsiConnection(connection):
```

Comment:
Would be better to use established naming convention in the form of "${scriptname}Connection"

Change it to `SpadenadeFixConnection` in that case?